tests/test_app/TestApp/Controller/PaginatorTestController.php

```diff
@@ -1,16 +0,0 @@
-<?php
```

Comment:
I have no idea why this file exists, but that class never gets instantiated, therefore i removed it.